src/KeyVault/KeyVault/Commands/GetAzureKeyVaultSecret.cs

```diff
@@ -209,7 +218,7 @@ public override void ExecuteCmdlet()
             if (!string.IsNullOrEmpty(Version))
             {
                 secret = DataServiceClient.GetSecret(VaultName, Name, Version);
-                WriteObject(secret);
+                WriteSecret(secret);
```

Comment:
Should we keep the same behaiour for the `deletedSecret` in line 241? I think it's reasonable.

Reply:
`deletedSecret` does not contain secret value, it's just metadata, so it is fine.